src/features/Character/components/CharacterDetail/CharacterDetail.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { VFC, memo } from 'react';
+import { Text } from '@chakra-ui/react';
+
+interface Props {
+
+  /** Detail title. */
+  readonly title: string;
+
+  /** Character detail. */
+  readonly detail: string;
+}
+
+/** Small character detail. */
+const CharacterDetailComponent: VFC<Props> = ({ title, detail }) => (
+  <Text>
+    <Text as="span" fontWeight="bold">
+      {title}
+      :
+      {' '}
+    </Text>
```

Comment:
Is it intended?

Reply:
Yes, this is intended. This way I add space after `:` symbol